auth-web/src/components/auth/create-account/GovmContactInfoForm.vue

```diff
@@ -97,48 +104,58 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Mixins } from 'vue-property-decorator'
-import { Action } from 'pinia-class'
+import { defineComponent, onMounted, reactive, ref, toRefs } from '@vue/composition-api'
 import ConfirmCancelButton from '@/components/auth/common/ConfirmCancelButton.vue'
 import NextPageMixin from '@/components/auth/mixins/NextPageMixin.vue'
 import Steppable from '@/components/auth/common/stepper/Steppable.vue'
-import { User } from '@/models/user'
 import { useUserStore } from '@/stores/user'
 
-@Component({
+export default defineComponent({
+  name: 'GovmContactInfoForm',
   components: {
     ConfirmCancelButton
-  }
-})
-export default class GovmContactInfoForm extends Mixins(NextPageMixin, Steppable) {
-  @Action(useUserStore) public getUserProfile!: (identifer: string) => User
+  },
+  mixins: [NextPageMixin, Steppable],
+  emits: ['final-step-action'],
+  setup (props, { root, emit }) {
+    const userStore = useUserStore()
+    const state = reactive({
+      emailAddress: '',
+      confirmedEmailAddress: '',
+      userProfile: null
+    })
 
-  public emailAddress = ''
-  public confirmedEmailAddress = ''
-  // @Prop({ default: false }) isStepperView: boolean
+    const formRef = ref(null)
 
-  $refs: {
-    form: HTMLFormElement
-  }
+    onMounted(async () => {
+      state.userProfile = await userStore.getUserProfile('@me')
+      state.emailAddress = state.userProfile?.email || ''
+      state.confirmedEmailAddress = state.userProfile?.email || ''
+    })
 
-  public async mounted () {
-    await this.getUserProfile('@me')
-    this.emailAddress = this.userProfile?.email || ''
-    this.emailAddress = this.confirmedEmailAddress = this.userProfile?.email || ''
-  }
-  // email is readonly to show. no need to save
-  @Emit('final-step-action')
-  public createAccount () {
-  }
+    const createAccount = () => {
+      // email is readonly to show. no need to save
+      emit('final-step-action')
+    }
 
-  public cancel () {
-    this.$router.push('/')
-  }
+    const cancel = () => {
+      root.$router.push('/')
+    }
+
+    const goBack = () => {
+      // Vue 3 - get rid of MIXINS and use the composition-api instead.
+      (props as any).stepBack()
```

Comment:
Typescript couldn't find the function on the prop, if I did it like

`props.stepBack()`